app/collections/manager.cjsx

```diff
@@ -17,7 +17,12 @@ module?.exports = React.createClass
     subject: React.PropTypes.object
     project: React.PropTypes.object
 
+  logModalChange: (clickedItem) ->
+    @props.contextRef?.geordi?.logEvent
```

Comment:
Could do, but we'd still need to call `geordi.makeHandler` somehow and `geordi` is undefined if passed by context.